packages/builder/sample-project/algob.config.js

```diff
@@ -1,9 +1,20 @@
+import { mkAccounts } from "algob";
+
+let accounts = mkAccounts([{
+  addr: "KFMPC5QWM3SC54X7UWUW6OSDOIT3H3YA5UOCUAE2ABERXYSKZS5Q3X5IZY",
+  mnemonic: "call boy rubber fashion arch day capable one sweet skate outside purse six early learn tuition eagle love breeze pizza loud today popular able divide"
+}]);
+
+let defaultCfg = {
+  host: "http://localhost",
+  port: 8080,
+  token: "$ALGORAND_DATA/algod.token",
```

Comment:
Do you expect that every developer will have this environment variable?

Reply:
no, but coding any other path makes less sense.

Reply:
What about a relative path to a file? What does this file contain? Isn't it possible to have a variable as for `accounts` array?

Reply:
this is something different. It's not related to accounts. It's needed for the client authorization to the node. 
PS: How did you connect to a private node without accessing / looking at the token file (that was one of the first tasks).

other solution is `<path to your token file>/algod.token` - which is not better.

Reply:
I don't remember, I simply used `goal` to run everything. So it should've handled that itself.